src/Global.purs

```diff
@@ -19,3 +19,6 @@ foreign import readInt :: Int -> String -> Number
 
 -- | Parse a floating point value from a `String`
 foreign import readFloat :: String -> Number
+
+-- | uri encoding
```

Comment:
To be more specific, I'd say this "Encodes a URI _component_", since it escapes joining characters, unlike `escapeURI`.